docs/EcoFacet.md

```diff
@@ -0,0 +1,139 @@
+  - Bridges tokens using Eco Protocol without performing any swaps
+- `function swapAndStartBridgeTokensViaEco(BridgeData memory _bridgeData, LibSwap.SwapData[] calldata _swapData, EcoData calldata _ecoData)`
+  - Performs swap(s) before bridging tokens using Eco Protocol
+
+## Eco Specific Parameters
+
+The methods listed above take a variable labeled `_ecoData`. This data is specific to Eco Protocol and is represented as the following struct type:
+
+```solidity
+/// @dev Eco specific parameters
+/// @param receiverAddress Address that will receive tokens on destination chain
+/// @param nonEVMReceiver Destination address for non-EVM chains (bytes format)
+/// @param prover Address of the prover contract for validation
+/// @param rewardDeadline Timestamp for reward claim eligibility
+/// @param solverReward Reward amount for the solver (native or ERC20 depending on sendingAssetId)
+/// @param encodedRoute Encoded route data containing destination chain routing information
+struct EcoData {
+  address receiverAddress;
+  bytes nonEVMReceiver;
+  address prover;
+  uint64 rewardDeadline;
+  uint256 solverReward;
+  bytes encodedRoute;
+}
+```
+
+### Address Parameters Usage
+
+The `receiverAddress` and `nonEVMReceiver` parameters serve different purposes:
+
+- **For EVM destination chains**: Use `receiverAddress` (standard Ethereum address format). Leave `nonEVMReceiver` empty.
+- **For non-EVM destination chains** (Solana, Tron, etc.):
+  - Set `receiverAddress` to `address(0)` or leave unchanged
+  - Set `bridgeData.receiver` to `NON_EVM_ADDRESS` constant
+  - Provide the destination address in `nonEVMReceiver` as bytes
+
+**Important**: Only one of these should be used per transaction. Never provide both values simultaneously.
+
+Examples:
+
+```solidity
+// EVM to EVM bridge
+ecoData.receiverAddress = 0x123...;  // Actual receiver address
+ecoData.nonEVMReceiver = "";         // Empty bytes
+
+// EVM to Solana bridge
+bridgeData.receiver = NON_EVM_ADDRESS;  // Special constant
+ecoData.receiverAddress = address(0);   // Not used
+ecoData.nonEVMReceiver = solanaAddressBytes;  // Actual Solana address
+```
+
+### Important Notes
+
+- **Solver Reward**:
+  - For native token transfers: The entire amount (bridge amount + solver reward) must be sent as `msg.value`
+  - For ERC20 transfers: The solver reward is included in the deposited/swapped amount and the facet handles the split internally
+- **Non-EVM Destinations**: For non-EVM chains (like Tron or Solana), set `receiver` in `BridgeData` to `NON_EVM_ADDRESS` (constant from src/Helpers/LiFiData.sol) and provide the destination address in `nonEVMReceiver`.
+- **Native Token Bridging**: When bridging native tokens, the bridge amount plus the solver reward must be sent as `msg.value`.
+- **ERC20 Token Bridging**: For ERC20 tokens, the facet will automatically approve the Eco Portal to spend the total amount (bridge amount + solver reward).
```

Comment:
please review this whole paragraph. It seems like it contains duplicate information e.g.:

1) **Non-EVM Destinations** >> is duplicate to paragraph above
2) **Native Token Bridging** >> is duplicate to "Solver Reward"